core/src/main/java/org/apache/calcite/sql/SqlUtil.java

```diff
@@ -417,24 +418,52 @@ public static void unparseSqlIdentifierSyntax(
     writer.endList(frame);
   }
 
+  private static void unparseSetOpSyntax(
```

Comment:
why is the function named `unparseSetOpSyntax`? The `Syntax` seems redundant, in which case it would be `unparseSetOp`. But then I see that it's unparsing a `SqlSelect`, not a set op. So I'm thoroughly confused.

Reply:
Sorry, the previous naming was really inappropriate. 
I renamed it to `unparseBinaryOperand`(Write parentheses for special operands in it)